### DISCUSSION TITLE
Grid columns issue: row >* overriding the width of col-lg-9 in Bootstrap 5

### DISCUSSION THREAD

**Discussion by nainesh279** [twbs/bootstrap#41077] [Q&A]
row { 
   @include make-row(); 
  
   > * { 
     @include make-col-ready(); 
   } 
 } 

Above css is overriding the % width of col-* classes and making it 100%, which is causing issue in using the Bootstrap Grid

**Comment by benabraham** [comment]
I don’t see a point posting the same question again. 

You have an answer in the original question: #41064
